spec/integration/foods_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Foods', type: :system, js: true do
+  before :all do
+    @user = User.create(name: 'Test User', email: 'user@mail.com', password: 'password')
+    @banana = Food.create(name: 'Banana', measurement_unit: 'g', price: 0.10, quantity: 100, user: @user)
+    @apple = Food.create(name: 'Apple', measurement_unit: 'units', price: 0.20, quantity: 100, user: @user)
+  end
+
+  before :each do
+    visit new_user_session_path
+    fill_in 'Email', with: @user.email
+    fill_in 'Password', with: 'password'
+    click_button 'Log in'
+
+    visit foods_path
+  end
+
+  it 'shows the food page information' do
+    expect(page).to have_content('Banana')
+    expect(page).to have_content('Apple')
+
+    expect(page).to have_content('$0.1')
+    expect(page).to have_content('$0.2')
+
+    expect(page).to have_content(@banana.measurement_unit)
+    expect(page).to have_content(@apple.measurement_unit)
```

Comment:
- [ ] As requested by the previous reviewer below tests are failing, Please make sure to fix them and all tests should pass without any errors. (I also tried running each of these tests individually But still failed. If they're passing on your machine, I would suggest you attach a screenshot in the comment so the next CR will approve the PR).

![image](https://github.com/Timbar09/RecipePlug/assets/83803414/cf209195-1a04-47f2-ba62-739792bae8b6)